code/modules/cm_marines/vehicle_part_fabricator.dm

```diff
@@ -149,7 +157,7 @@
 		var/index = params["index"]
```

Comment:
I doubt this is bug caused by this PR (more likely #7084 ) ; but in testing this I am able to get the index == null if I queue up items and cancel them: 
![image](https://github.com/user-attachments/assets/225356f2-400a-4ebd-b1de-bbe8faa05378)

Seems fairly easy to reproduce by queueing say 5 things, and then spam clicking cancel on whatever is item 2 until you get an exception.